package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "droidconke2022reactnative",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "expo": "~46.0.9",
+    "expo-status-bar": "~1.4.0",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-native": "0.69.5",
+    "react-native-web": "~0.18.7"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "@types/react": "~18.0.14",
+    "@types/react-native": "~0.69.1",
+    "typescript": "~4.3.5"
+  },
+  "private": true,
+  "engines": {
+    "node": "14.7.0"
```

Comment:
```suggestion
    "node": "14.7.0",
    "npm": "'^7.0.0"
```

Apologies for the back and forth, might you also consider adding a version for npm?

Reason: npm v6 and less uses npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. We can also lock it to a version thats higher than 7.0.0

A sample of the warning:

<img width="1340" alt="image" src="https://user-images.githubusercontent.com/5811391/188245494-834de3e4-a75f-48e1-8d73-886f1e7eb080.png">

Reply:
That's okay, lemme do that right away!